server/src/com/cloud/tags/TaggedResourceManagerImpl.java

```diff
@@ -297,16 +297,46 @@ public String getUuid(String resourceId, ResourceObjectType resourceType) {
     @ActionEvent(eventType = EventTypes.EVENT_TAGS_DELETE, eventDescription = "deleting resource tags")
     public boolean deleteTags(List<String> resourceIds, ResourceObjectType resourceType, Map<String, String> tags) {
         Account caller = CallContext.current().getCallingAccount();
+        List<String> uuids = new ArrayList<String>();
```

Comment:
deleteTags() is already to big. Please factor out the new code in a separate set of methods.